### PR TITLE
Creates /osm page

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Creates /osm page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/telco/osm
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check modal form works and that the static form works is JS is disbaled.
- Compare against [copy-doc](https://docs.google.com/document/d/1LQim0WRVdDZE9-ToFKSD52O7kgurMoytrgDDAPYnug8/edit)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/10064

### CODE REVIEW

**Comment by petesfrench** [comment]
Demo: https://ubuntu-com-10068.demos.haus/telco/osm

**Comment by SirSamTumless** [comment]
@WajeehaHamid Please provide links to the images in the copy doc.

File: templates/telco/osm/index.html
Comment (bethcollins92) [inline templates/telco/osm/index.html:69]: Line 69 no longer exists in the copy doc

File: templates/telco/osm/index.html
Comment (bethcollins92) [inline templates/telco/osm/index.html:77]:
```suggestion
        <a href="https://charmed-osm.com/" class="p-link--external">Read more</a>
```

File: templates/telco/osm/index.html
Comment (bethcollins92) [inline templates/telco/osm/index.html:121]:
```suggestion
        <a class="p-link--external" href="https://juju.is/docs/sdk">Understand Juju and charms</a>
```

File: templates/telco/osm/index.html
Comment (bethcollins92) [inline templates/telco/osm/index.html:191]:
```suggestion
      <a href="https://charmed-osm.com/osm-hackfests" class="p-link--external">Discover our contributions</a>
```

File: templates/telco/osm/index.html
Comment (bethcollins92) [inline templates/telco/osm/index.html:262]:
```suggestion
            <a href="/engage/telco-network-functions-whitepaper">Building and orchestrating network functions&nbsp;&rsaquo;</a>
```

File: templates/telco/osm/index.html
Comment (bethcollins92) [inline templates/telco/osm/index.html:5]:
```suggestion
{% block meta_description %}Canonical Charmed OSM, open source,carrier grade and supported orchestrator for telecommunication service providers. {% endblock %}
```

File: templates/telco/osm/index.html
Comment (bethcollins92) [inline templates/telco/osm/index.html:7]:
```suggestion
{% block meta_copydoc %}https://docs.google.com/document/d/1LQim0WRVdDZE9-ToFKSD52O7kgurMoytrgDDAPYnug8/edit#% endblock meta_copydoc %}
```

File: templates/telco/osm/index.html
Comment (bethcollins92) [inline templates/telco/osm/index.html:53]: Only cols are allowed to be direct children of rows, this isn't within the col div - could you take it out of the row it's in and put it in a `<div class="u-fixed-width">`?  (Same with the others please)

File: templates/telco/osm/index.html
Comment (bethcollins92) [inline templates/telco/osm/index.html:283]: can you add `attrs={"class": "p-inline-images__logo"},` to all these images?

File: templates/telco/osm/index.html
Comment (bethcollins92) [inline templates/telco/osm/index.html:453]:
```suggestion
      <a href="/telco/osm/contact-us" class="p-button--positive js-invoke-modal">Let's talk</a>
```

File: templates/telco/osm/index.html
Comment (bethcollins92) [inline templates/telco/osm/index.html:471]: are you still waiting on the formID?

File: templates/shared/forms/interactive/osm-contact-us.html
Comment (bethcollins92) [inline templates/shared/forms/interactive/osm-contact-us.html:35]: We use a text input for other like this:
`              <input type="text" id="infrastructure-preferences-other" class="js-other-container__input" placeholder="Please specify" style="opacity: 0; margin-top: .25rem;" aria-label="Other architecture">`

File: templates/shared/forms/interactive/osm-contact-us.html
Comment (bethcollins92) [inline templates/shared/forms/interactive/osm-contact-us.html:77]: same for this and all the other "other" options

File: templates/shared/forms/interactive/osm-contact-us.html
Comment (bethcollins92) [inline templates/shared/forms/interactive/osm-contact-us.html:117]: These yes and no options should be radio buttons not checkboxes, search for radio buttons on ubuntu.com to see an example

**Review by bethcollins92 — CHANGES_REQUESTED** [review]
Nice work Waiting on images to be added before final review but some changes while we wait. Seems like a lot of comments but it's only small fixes - let me know if you have any questions 👍🏼  (have you been given the correct marketo form ID for this contact page?)

File: templates/shared/forms/interactive/osm-contact-us.html
Comment (petesfrench) [reply templates/shared/forms/interactive/osm-contact-us.html:77]: The same as what?

**Comment by WajeehaHamid** [comment]
> @WajeehaHamid Please provide links to the images in the copy doc.

I provided all the links I had. Thanks

File: templates/shared/forms/interactive/osm-contact-us.html
Comment (bethcollins92) [reply templates/shared/forms/interactive/osm-contact-us.html:77]: us the specific "other containter" mentioned [here](https://github.com/canonical-web-and-design/ubuntu.com/pull/10068#discussion_r677531279)

Comment (petesfrench) [reply templates/shared/forms/interactive/osm-contact-us.html:77]: I see, thanks

File: templates/telco/osm/index.html
Comment (sowasred2012) [inline templates/telco/osm/index.html:198]: As discussed in our call just now - the `u-fixed-width` div should only contain the `h2`, and the `.row.p-divider` element should come up one indentation level to sit alongside the `u-fixed-width`, so that there isn't extra left and right padding on the divider block.

File: templates/telco/osm/index.html
Comment (bethcollins92) [inline templates/telco/osm/index.html:85]:
```suggestion
        <a class="p-link--external" href="https://charmed-osm.com/">Read more</a>
```

File: templates/telco/osm/index.html
Comment (bethcollins92) [reply templates/telco/osm/index.html:62]: Do you want some help with this? we can jump on a call to show you what I mean if you like

File: templates/telco/osm/index.html
Comment (bethcollins92) [inline templates/telco/osm/index.html:199]:
```suggestion
      <a class="p-link--external" href="https://charmed-osm.com/osm-hackfests">Discover our contributions</a>
```

File: templates/telco/osm/index.html
Comment (bethcollins92) [inline templates/telco/osm/index.html:271]:
```suggestion
          <a href="/engage/telco-network-functions-whitepaper">Building and orchestrating network functions&nbsp;&rsaquo;</a>
```

**Review by bethcollins92 — CHANGES_REQUESTED** [review]

File: templates/telco/osm/index.html
Comment (petesfrench) [reply templates/telco/osm/index.html:62]: Are you referring to the hr tag?

File: templates/shared/forms/interactive/osm-contact-us.html
Comment (bethcollins92) [inline templates/shared/forms/interactive/osm-contact-us.html:129]: this should be a text area

File: templates/shared/forms/interactive/osm-contact-us.html
Comment (bethcollins92) [inline templates/shared/forms/interactive/osm-contact-us.html:1]: Currently this form is throwing a 403 error on submit. Could you try this fix https://github.com/canonical-web-and-design/ubuntu.com/pull/10097/files ? There was this issue with another contact form and this fixed it

File: templates/telco/osm/index.html
Comment (bethcollins92) [inline templates/telco/osm/index.html:195]:
```suggestion
    <p class="p-heading--four">52 Participants, more than 100 Network Functions onboarding</p>
```

**Review by bethcollins92 — APPROVED** [review]
Yaay, looking good! Thanks for addressing those comments. Just one TINY change then you're good to go once you have a design +1 (maybe but a Pdm label on this too to get them to review it before you merge)

File: templates/telco/osm/index.html
Comment (petesfrench) [reply templates/telco/osm/index.html]: Nice spot! Thanks for all the feedback!

**Comment by petesfrench** [comment]
@WajeehaHamid Can you take a look at this and give it a PdM +1 label if it all looks good.

**Comment by bethcollins92** [comment]
@WajeehaHamid could you take a look at this and add the pdm +1 if you're happy? thanks

**Comment by WajeehaHamid** [comment]
> @WajeehaHamid could you take a look at this and add the pdm +1 if you're happy? thanks

Hi @bethcollins92 
Everything is perfect. Can we please remove the section "Participation in OSM Hackfests" from the end? I just checked it got repeated in two pages.
Remove the section before this line "Discover our contributions"

**Comment by petesfrench** [comment]
@WajeehaHamid I have removed that section if you want to take another look and give it a pdm+1 if it looks good

**Comment by petesfrench** [comment]
I received a verbal pdm +1 from @WajeehaHamid so will merge this.

**Comment by WajeehaHamid** [comment]
I did that on Github. Thanks

**Pull request merged**
